src/types.ts

```diff
@@ -0,0 +1,33 @@
+import { CumulusMessage, ReplaceConfig } from '@cumulus/types/message';
+import { Context } from 'aws-lambda';
+import { ChildProcessWithoutNullStreams } from 'child_process';
+
+export interface CumulusMessageAdapterError { stderrBuffer: string }
+
+export interface InvokeCumulusMessageAdapterResult {
+  cmaProcess: ChildProcessWithoutNullStreams,
+  errorObj: CumulusMessageAdapterError
+}
+
+export interface LoadNestedEventInput {
+  input: unknown,
+  config: unknown,
+  messageConfig?: unknown
+}
+export interface CumulusMessageWithPayload extends CumulusMessage {
+  payload: {
+    granules?: { granuleId: string }[] | undefined
```

Comment:
Because this has a question mark, the ` | undefined` is not needed.

Reply:
ee25df1